src/runtime/components/Sections/index.vue

```diff
@@ -2472,8 +2510,35 @@ const getComponentSetup = async (sectionName, sectionType) => {
     return setupData
   }
 }
-const getComponent = (sectionName, sectionType) => {
+const sectionSEO = ref({
+  title: "",
+  description: "",
+  image: ""
+})
+const seoManagement = (view, hooksJs) => {
+  try {
+    if (view && pageMetadata.value.seo && pageMetadata.value.seo[view.id] === true) {
+      if (hooksJs && hooksJs['seo_management']) {
+        const seo = JSON.parse(JSON.stringify(hooksJs['seo_management'](view, i18n.locale.value)))
+        if (seo.title && !sectionSEO.value.title) {
+          sectionSEO.value.title = seo.title
+          computedTitle.value = seo.title
+        }
+        if (seo.description && !sectionSEO.value.description) {
+          sectionSEO.value.description = seo.description
+          computedDescription.value = seo.description
+        }
+        if (seo.image && !sectionSEO.value.image) {
+          sectionSEO.value.image = seo.image
+          computedImage.value = seo.image
```

Comment:
why are you handpicking the tags ? why don't you just relay a tag object that contains the fields used by the metadata module ?

Reply:
This is because title, description and image are separate computed properties (to have them picked up by the useHead composable). And also this allows setting the title of a section 1 (first in order) and the description of section 2 (second in order) in case section 1 doesn't return a description

Reply:
You can read the fields of an object inside the useHead composable, can't you ?

As for the second point, this is solved by merging the objects and giving precedence to the correct one